scipy/stats/_stats_py.py

```diff
@@ -7214,15 +7226,11 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
     -----
     This test is invalid when the observed or expected frequencies in each
     category are too small.  A typical rule is that all of the observed
-    and expected frequencies should be at least 5. According to [3]_, the
-    total number of samples is recommended to be greater than 13,
+    and expected frequencies should be at least 5. According to [2]_, the
+    total number of observations is recommended to be greater than 13,
     otherwise exact tests (such as Barnard's Exact test) should be used
     because they do not overreject.
```

Comment:
```suggestion
    otherwise exact tests (such as Barnard's Exact test or a permutation test) should be used
    because they do not overreject.
```
Might as well point them towards the more general right way to do it (Barnard's test is for 2x2 tables). Note that you can't just permute the counts, but rather the categorical data the counts came from; this should point them in the right direction.

Reply:
Sure, you're welcome to open a PR with that information. This PR does not intend to adjust much more than necessary to document the new parameter and why it might be used.